app/views/layouts/_tracking_scripts.html.erb

```diff
@@ -78,17 +78,31 @@ twq('track','PageView');
 </script>
 <!-- End Twitter universal website tag code -->
 
+<% if request.original_fullpath.start_with? '/signals' %>
+<!-- Hotjar Tracking Code for https://www.coinfi.com/signals -->
+<script>
+(function(h,o,t,j,a,r){
+  h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+  h._hjSettings={hjid:1034618,hjsv:6};
```

Comment:
Is the hotjar id the only difference in these two blocks? Is it better to just customize that with a variable?

Reply:
It is, great suggestion!